modules/fcrepo/main.tf

```diff
@@ -0,0 +1,150 @@
+# Actual direct beanstalk configuration for the fedora core.
+resource "aws_elastic_beanstalk_application" "fedora" {
+  name        = "${var.StackName}"
+  description = "Fcrepo service"
+}
+
```

Comment:
When we go to build the whole stack, do we want each beanstalk service to get its own application or wrap them all under one shared application? I've been building the solrcloud configs as tho we were going to build one application outside of the beanstalk modules.

Reply:
I honestly don't know the pros and cons there.  I'm going off of what seems like the most literal port of the existing configuration, but I don't know enough about the EB applications to really have a good answer.